packages/components/src/tooltip/stories/index.story.tsx

```diff
@@ -57,3 +57,16 @@ KeyboardShortcut.args = {
 		ariaLabel: shortcutAriaLabel.primaryShift( ',' ),
 	},
 };
+
+/**
+ * Nested `Tooltip` components simply ignore the tooltip part, and only
```

Comment:
This comment seems unfinished.

Reply:
Well spotted, I updated it with the same text that I added to the README.